### PR TITLE
Added threshold notifications and min qty for Asset models

### PR DESCRIPTION
**Pull request by Godmartinz**
# Description
This adds a minimum quantity threshold for Asset Models. Also updates the notifcations to include Asset Models.

image
image

Fixes #sc-19826

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* PHP version:
* MySQL version
* Webserver version
* OS version

# Checklist:

- [ ] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [ ] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/Http/Transformers/ActionlogsTransformer.php [hunk 1]
```diff
@@ -122,7 +121,9 @@ public function transformActionlog (Actionlog $actionlog, $settings = null)
             'log_meta'          => ((isset($clean_meta)) && (is_array($clean_meta))) ? $clean_meta: null,
             'action_date'   => ($actionlog->action_date) ? Helper::getFormattedDateObject($actionlog->action_date, 'datetime'): Helper::getFormattedDateObject($actionlog->created_at, 'datetime'),
         ];
+        //\Log::info("Clean Meta is: ".print_r($clean_meta,true));
```
Comment (snipe) [inline app/Http/Transformers/ActionlogsTransformer.php]: Can we pull these commented out lines?

**Review by snipe — CHANGES_REQUESTED** [review]
I think we also need to add `min_amt` to the `fillable` array in the `AssetModel` model since we use ` $assetmodel->fill($request->all());` in the API, otherwise additions/updates via the API won't be able to access that value. (Unless it's already in there and I just missed it.)

Otherwise this looks pretty solid.

Not for this PR, but we should also see about adding models to the inventory alert emails.

**Comment by Godmartinz** [comment]
`min_amt` is already added to `fillable` in the `AssetModel` model. Also, for the Inventory Alert email, where would the asset_model quantity info need to be sent? My understanding is that it goes to the `Helper:checkLowInventory()`

**Comment by snipe** [comment]
Ah, you're right, my bad. The GH diff made it look weird. If you can yank those commented lines out, this looks good to go.

**Comment by snipe** [comment]
Looks like we have a merge conflict now

**Comment by snipe** [comment]
Nice, quick as a bunny :)

**Comment by Godmartinz** [comment]
Also, running the low inventory command recognizes and send out an email for asset models. 
image

**Comment by snipe** [comment]
The chipper test is failing on this, but I think it's related to an earlier merge.

**Pull request merged**